tests/components/homewizard/snapshots/test_diagnostics.ambr

```diff
@@ -261,13 +261,11 @@
         'api_version': 'v1',
         'firmware_version': '3.06',
         'product_name': 'kWh meter',
-        'product_type': 'SDM230-WIFI',
+        'product_type': 'SDM230-wifi',
         'serial': '**REDACTED**',
       }),
       'state': None,
-      'system': dict({
-        'cloud_enabled': True,
-      }),
+      'system': None,
```

Comment:
Is this unavailable for the SDM230?

Reply:
Yup, this is a bug in the current implementation, the tests now reveal that.
Will fix this in a later iteration.